test/test-normalize-hotkey.js

```diff
@@ -0,0 +1,36 @@
+import {normalizeHotkey} from '../dist/index.js'
+
+describe('normalizeHotkey', () => {
+  it('should exist', () => {
+    assert.isDefined(normalizeHotkey)
+  })
+
+  const tests = [
+    // Base case control tests
+    ['a', 'a'],
+    ['Control+a', 'Control+a'],
+    ['Meta+a', 'Meta+a'],
+    ['Control+Meta+a', 'Control+Meta+a'],
+    // Mod should be localized based on platform
+    ['Mod+a', 'Control+a', 'win'],
+    ['Mod+a', 'Meta+a', 'mac'],
+    ['Mod+A', 'Control+A', 'win'],
+    ['Mod+A', 'Meta+A', 'mac'], // TODO: on a mac upper-case keys are lowercased when Meta is pressed
+    ['Mod+9', 'Control+9', 'win'],
+    ['Mod+9', 'Meta+9', 'mac'],
+    ['Mod+)', 'Control+)', 'win'],
+    ['Mod+)', 'Meta+)', 'mac'], // TODO: on a mac upper-case keys are lowercased when Meta is pressed
```

Comment:
Note that the tests on line 18 and 22 are incorrect: the hotkey `Meta+A` won't match on Mac: instead it _should_ be `Meta+Shift+a` in the current  implementation. Likewise, `Meta+)` should be `Meta+Shift+0`.

I hope to address this in https://github.com/github/hotkey/issues/54